code/modules/antagonists/changeling/powers/fakedeath.dm

```diff
@@ -26,6 +33,9 @@
 	return TRUE
 
 /datum/action/changeling/fakedeath/proc/revive(mob/living/user)
+	if(!user || !istype(user))
+		message_admins("Warning! Ling revive user is invalid! Check runtime logs!")
+		log_game("Warning! Ling revive user is invalid! Check runtime logs!")

```

Comment:
Replace these with a return, no need to log or message admins and it doesn't actually stop a runtime.